### PR TITLE
Admin can manage businesses

### PR DESCRIPTION
**Pull request by dtinianow**

### CODE REVIEW

**Comment by Yoyo2Code** [comment]
Awesome idea nesting the params into a usable hash.

File: spec/features/user_can_cancel_order_spec.rb [hunk 1]
```diff
@@ -20,9 +20,9 @@
 
     visit property_path(property, business_name: property.business.slug)
 
-    find('#occupancy').find(:xpath, 'option[2]').select_option
-    fill_in :starting_date, with: "08/30/2016"
-    fill_in :end_date, with: "09/05/2016"
+    find('#booking_occupancy').find(:xpath, 'option[2]').select_option
```
Comment (Yoyo2Code) [inline spec/features/user_can_cancel_order_spec.rb]: Great play using xpath instead of searching or within for finding the selection.

**Comment by Automatic365** [comment]
Every works pretty cleanly here. good job

File: app/views/properties/show.html.erb [hunk 2]
```diff
@@ -5,18 +5,18 @@
   <div class='col-md-4 col-md-offset-1 well property-image'>
     <img src=" <%= @property.image_path %>"  /><br><br>
   </div>
-
   <div class='col-md-5 property-details'>
-    
+
     <h4><span class="glyphicon glyphicon-map-marker"></span>
-      Location: 
+      Location:
```
Comment (Yoyo2Code) [inline app/views/properties/show.html.erb]: Is there a specific reason for the new line here?

File: app/views/properties/show.html.erb [hunk 3]
```diff
@@ -5,18 +5,18 @@
...
+
```
Comment (Yoyo2Code) [inline app/views/properties/show.html.erb]: Is there a reason for a new line here?

**Pull request merged**